.travis.yml

```diff
@@ -46,6 +46,6 @@ matrix:
       - pip3 install scipy
 
 script: 
-  - mkdir -p precice_future && echo -e "from setuptools import setup\nsetup(name='precice_future')" > precice_future/setup.py
-  - $PY -m pip install ./precice_future/
+  - mkdir -p precice && echo -e "from setuptools import setup\nsetup(name='precice', version="2.0.0")" > precice/setup.py
```

Comment:
preCICE version here is 2.0.0, do we wait till v2 release or merge this right now and bump the version later?

Reply:
The version of the fenics adapter that we want to merge will only be compatible with preCICE 2.0.0, since otherwise the names and everything of the python bindings do not match. Therefore, I would keep the 2.0.0 here. Otherwise, we suggest a compatibility with older, incompatible versions.

Reply:
Oh okay yes then I agree that we keep 2.0.0 here!